Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -261,9 +261,9 @@ public function remove(string $entryIdentifier): bool
                     clearstatcache(true, $cacheEntryPathAndFilename);
                     return $result;
                 }
-            } catch (\Exception $e) {
+            } catch (\Exception) {
             }
-            usleep(rand(10, 500));
+            usleep(random_int(10, 500));
```

Comment:
do we need the transformation of all the random functions?

Reply:
I would say using real randomness is generally a good practice. So i would say by default yes but we might chose to use rand in places for performance reasons.

Reply:
I can't say much about it just wanted to point it out ;)